cli/cmd/cluster_addon_flags.go

```diff
@@ -0,0 +1,28 @@
+package cmd
+
+import (
+	"time"
+
+	"github.com/spf13/cobra"
+)
+
+type clusterAddonArgs struct {
+	outputFormat string
+}
+
+type clusterAddonCreateArgs struct {
+	clusterID    string
+	waitDuration time.Duration
+	dryRun       bool
+}
+
+func clusterAddonFlags(cmd *cobra.Command, args *clusterAddonArgs) error {
+	cmd.Flags().StringVar(&args.outputFormat, "output", "table", "The output format to use. One of: json|table|wide (default: table)")
+	return nil
+}
+
+func clusterAddonCreateFlags(cmd *cobra.Command, args *clusterAddonCreateArgs) error {
+	cmd.Flags().DurationVar(&args.waitDuration, "wait", 0, "Wait duration for addon to be ready (leave empty to not wait)")
+	cmd.Flags().BoolVar(&args.dryRun, "dry-run", false, "Dry run")
+	return nil
+}
```

Comment:
I guess the idea is we should be able to reuse this for other addons later. It just makes it less readable imo.

Reply:
its tricky because i see an issue with the current cli and consistency in flags. im removing this in favor of consistency in code, but i like how other clis ive seen reuse common code for flags.